assets/src/edit-story/components/popup/index.js

```diff
@@ -131,20 +126,18 @@ function Popup({
 
   useLayoutEffect(() => {
     setMounted(true);
-    positionPopup();
-
-    // Adjust the position when scrolling or resizing.
-    window.addEventListener('resize', positionPopup);
+    // Adjust the position when scrolling.
     document.addEventListener('scroll', positionPopup, true);
     return () => {
-      window.removeEventListener('resize', positionPopup);
       document.removeEventListener('scroll', positionPopup, true);
     };
   }, [positionPopup]);
 
-  useEffect(() => {
-    positionPopup();
-  }, [placement, spacing, anchor, dock, popup, isOpen, positionPopup]);
+  useLayoutEffect(positionPopup, [positionPopup, isOpen]);
```

Comment:
We need to run it an extra time when it opens because we don't have access to the popover DOM node to obtain size of the element and calculate its proper horizontal & vertical boundaries

Reply:
Can't this line go in the other `useLayoutEffect` or does it not re-mount often enough (because `isOpen` isn't a dependency)?

Reply:
yea doesn't remount often enough because `isOpen` isn't a dependency.

We can probably smoosh them together tho. might make sense to only have a scroll listener binded if popover is open. I'll check this out and make sure it doesn't cause any race conditions. Should be a little lighter weight to only update an the open popover on scroll too